src/Microsoft.CodeAnalysis.Razor/DefaultTagHelperDescriptorFactory.cs

```diff
@@ -44,9 +31,8 @@ internal class DefaultTagHelperDescriptorFactory
                 .WithGlobalNamespaceStyle(SymbolDisplayGlobalNamespaceStyle.Omitted)
                 .WithMiscellaneousOptions(SymbolDisplayFormat.FullyQualifiedFormat.MiscellaneousOptions & (~SymbolDisplayMiscellaneousOptions.UseSpecialTypes));
 
-        public DefaultTagHelperDescriptorFactory(Compilation compilation, bool designTime)
+        public DefaultTagHelperDescriptorFactory(Compilation compilation)
```

Comment:
Will need to react to this in our layer that returns `TagHelperDescriptor`s to tooling OR will need to make a WTE change. I'd prefer the first.